packages/core/src/connectors/utilities/aliyun.ts

```diff
@@ -51,20 +51,40 @@ export interface PublicParameters {
   SignatureNonce?: string;
 }
 
-const commonParameters = {
+// Check SMS public parameters and default value @doc https://help.aliyun.com/document_detail/29440.html
+export interface TextPublicParameters {
+  Signature?: string;
+  AccessKeyId: string;
+  Format?: string; // 'json' or 'xml', default: 'json'
+  RegionId?: string;
+  SignatureMethod?: string;
+  SignatureNonce?: string;
+  SignatureVersion?: string;
+  Timestamp?: string;
+  Version?: string;
+}
+
+const mailCommonParameters = {
   Version: '2015-11-23',
   Format: 'json',
   SignatureVersion: '1.0',
   SignatureMethod: 'HMAC-SHA1',
 };
 
+const textCommonParameters = {
```

Comment:
no need to create a `textCommonParameters`, just remove `Version` from commonParameters